<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_SerialPort" stretch="1,1,6">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_SerialPort">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Serial Port</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_SerialPortGroup" stretch="2,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Label">
             <item>
              <widget class="QLabel" name="label_Port">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Port :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_BaudRate">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>BaudRate :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PacketRate">
               <property name="text">
                <string>PacketRate :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Coordinate">
               <property name="text">
                <string>Coordinate  :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Parameter">
             <item>
              <widget class="MyComboBox" name="comboBox_Port">
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentText">
                <string/>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::InsertAtBottom</enum>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_BaudRate">
               <property name="currentIndex">
                <number>4</number>
               </property>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>230400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>460800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>921600</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_PacketRate">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_Coordinate">
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>NED (North-East-Down)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ENU (East-North-Up)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Connect">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3DCube"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_ChartView" stretch="0,0,0">
        <item>
         <widget class="QCustomPlot" name="customPlot_accel" native="true"/>
        </item>
        <item>
         <widget class="QCustomPlot" name="customPlot_gyro" native="true"/>
        </item>
        <item>
         <widget class="QCustomPlot" name="customPlot_attitude" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionSet_Axis_Range">
   <property name="text">
    <string>Set Axis Range</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
